### PR TITLE
split the layout into components, displayed hotels from the array

### PR DESCRIPTION
**Pull request by M-fkstv**

### CODE REVIEW

File: src/App.test.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { render, screen } from "@testing-library/react";
-import App from "./App";
+import App from "./Components/App/App";
 
 test("renders learn react link", () => {
```
Comment (5tronciy) [inline src/App.test.js]: The test doesn't pass.

File: src/Components/Section/Section.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+export const Section = ({h2Text}) => (
+  <section className="homes">
+
+    <h2 className="h2-text">{h2Text}</h2>
+    <div className="homes__examples">
+
+      { hotels.map((item) => (
+        <Fragment key={item.id}>
+          <HotelCard  {...item} />
+        </Fragment>
```
Comment (5tronciy) [inline src/Components/Section/Section.js]: There is no need for React.Fragment to return one element only.

File: src/Components/App/App.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+
+import "./App.css";
+
+import {Section} from '../Section';
+
+
+function App() {
+  return (
+
+      <Section h2Text={'Homes Guest Loves'} />
```
Comment (5tronciy) [inline src/Components/App/App.js]: 'h2Text' - the example of bad naming.

**Review by 5tronciy — CHANGES_REQUESTED** [review]
The layout doesn't meet the design. It should be the same as in the first module.
Use the Prettier, please. Your code is hard to read.

File: src/App.test.js (hunk 1 above)
Comment (M-fkstv) [reply src/App.test.js]: Corrected

File: src/Components/Section/Section.js (hunk 2 above)
Comment (M-fkstv) [reply src/Components/Section/Section.js]: Done

File: src/Components/App/App.js (hunk 3 above)
Comment (M-fkstv) [reply src/Components/App/App.js]: Corrected

**Comment by M-fkstv** [comment]
Made corrections to the project

File: src/Components/SliderButton/SliderButton.js [hunk 4]
```diff
@@ -0,0 +1,6 @@
+import React from "react";
+import "./SliderButton.css";
+
+export const SliderButton = ({ className }) => (
+  <div className={className}></div>
+);
```
Comment (5tronciy) [inline src/Components/SliderButton/SliderButton.js:6]: Do you need this component with reduced data?

**Review by 5tronciy — CHANGES_REQUESTED** [review]
Use all the data from the array.
For displaying all the cards in the carousel, you may use [swiper](https://www.npmjs.com/package/swiper) or any other lib of your choice.

File: src/Components/SliderButton/SliderButton.js (hunk 4 above)
Comment (M-fkstv) [reply src/Components/SliderButton/SliderButton.js:6]: Added swiper for displaying all the cards  from the array.

**Review by almuly — APPROVED** [review]

**Pull request merged**